frontend/src/app/person-edit/person-edit.component.spec.ts

```diff
@@ -145,76 +290,44 @@ describe('PersonEditComponent', () => {
-      expect(spouse.value).toBe('Jane Doe');
-      const entryDate = nativeElement.querySelector('#entryDate');
-      expect(entryDate.value).toBe('01/12/2016');
-      const housing: HTMLSelectElement = nativeElement.querySelector('#housing');
-      expect(housing.options[housing.selectedIndex].value).toBe(person.housing);
-      const housingSpace = nativeElement.querySelector('#housingSpace');
-      expect(+housingSpace.value).toBe(person.housingSpace);
-      const hostName = nativeElement.querySelector('#hostName');
-      expect(hostName.value).toBe(person.hostName);
-      const fiscalStatusUnknown = nativeElement.querySelector('#fiscalStatusUNKNOWN');
-      expect(fiscalStatusUnknown.checked).toBe(false);
-      const fiscalStatusNotTaxable = nativeElement.querySelector('#fiscalStatusNOT_TAXABLE');
-      expect(fiscalStatusNotTaxable.checked).toBe(false);
-      const fiscalStatusTaxable = nativeElement.querySelector('#fiscalStatusTAXABLE');
-      expect(fiscalStatusTaxable.checked).toBe(true);
-      const fiscalNumber = nativeElement.querySelector('#fiscalNumber');
-      expect(fiscalNumber.value).toBe('0123456789012');
-      const fiscalStatusUpToDate = nativeElement.querySelector('#fiscalStatusUpToDate');
-      expect(fiscalStatusUpToDate.checked).toBe(person.fiscalStatusUpToDate);
-      const healthCareCoverage: HTMLSelectElement = nativeElement.querySelector('#healthCareCoverage');
-      expect(healthCareCoverage.options[healthCareCoverage.selectedIndex].value).toBe(person.healthCareCoverage);
-      const healthCareCoverageStartDate = nativeElement.querySelector('#healthCareCoverageStartDate');
-      expect(healthCareCoverageStartDate.value).toBe('01/01/2017');
-      const healthInsurance: HTMLSelectElement = nativeElement.querySelector('#healthInsurance');
-      expect(healthInsurance.options[healthInsurance.selectedIndex].value).toBe(person.healthInsurance);
-      const healthInsuranceStartDate = nativeElement.querySelector('#healthInsuranceStartDate');
-      expect(healthInsuranceStartDate.value).toBe('02/02/2017');
-      const accompanying = nativeElement.querySelector('#accompanying');
-      expect(accompanying.value).toBe(person.accompanying);
-      const socialSecurityNumber = nativeElement.querySelector('#socialSecurityNumber');
-      expect(socialSecurityNumber.value).toBe(person.socialSecurityNumber);
-      const cafNumber = nativeElement.querySelector('#cafNumber');
-      expect(cafNumber.value).toBe(person.cafNumber);
-      const nationality = nativeElement.querySelector('#nationality');
-      expect(nationality.value).toBe(person.nationality.name);
-
-      lastName.value = 'Do';
-      lastName.dispatchEvent(new Event('input'));
-      nativeElement.querySelector('#save').click();
+      const tester = new PersonEditTester(TestBed.createComponent(PersonEditComponent));
+      tester.detectChanges();
+
+      expect(tester.firstName.value).toBe(person.firstName);
```

Comment:
Maybe a dumb idea, but a custom matcher would be nice: `expect(tester.firstName).toHaveValue(person.firstName)`